### ISSUE TITLE
google.maps undefined error

### ISSUE THREAD

**Issue by chicheongweng** [israelidanny/ion-google-place#62]
How should I get the google.maps initialized?

**Comment by RamyRais** [comment]
yes of course google map need to be initialized. You can use angular-google-maps 
http://angular-ui.github.io/angular-google-maps/#!/use
